apis/v1alpha1/route_types.go

```diff
@@ -35,6 +35,24 @@ const (
 	GatewayAllowSameNamespace GatewayAllowType = "SameNamespace"
 )
 
+const (
+	// AnnotationAppProtocol defines the protocol a Gateway should use for
+	// communication with a Kubernetes Service.
+	// This annotation must be applied to the `BackendPolicy` resource and will
+	// apply to all Service resources that are selected by
+	// `BackendPolicy.Spec.BackendRefs`. The annotation serves as a stop-gap
```

Comment:
It's likely worth clarifying that BackendRefs can select Service ports directly. So this annotation would apply to Service ports selected by BackendRefs.

Reply:
I did some minor rephrasing here. PTAL.